src/main/resources/unicode_map.csv

```diff
@@ -0,0 +1,159 @@
+Æ,AE
+Ð,D
+Ø,O
+æ,AE
+ð,D
+ø,O
+Đ,D
+đ,D
+Ħ,H
+ħ,H
+ı,i
+Ĳ,IJ
+ĳ,IJ
+ĸ,k
+Ŀ,L
+ŀ,L
+Ł,L
+ł,L
+ŉ,n
+Ŋ,N
+ŋ,N
+Œ,OE
+œ,OE
+Ŧ,T
+ŧ,T
+ƀ,b
+Ɓ,B
+Ƃ,b
+ƃ,b
+Ƅ,b
+ƅ,b
+Ƈ,C
+ƈ,C
+Ɖ,D
+Ɗ,D
+Ƌ,d
+ƌ,d
+Ǝ,E
+Ɛ,e
+Ƒ,F
+ƒ,F
+Ɠ,G
+ƕ,HV
+Ɩ,I
+Ɨ,I
+Ƙ,K
+ƙ,K
+ƚ,l
+Ɯ,W
+Ɲ,N
+ƞ,n
+Ɵ,O
+Ƣ,OI
+ƣ,OI
+Ƥ,P
+ƥ,P
+Ʀ,R
+Ƨ,S
+ƨ,S
+ƪ,l
+ƫ,t
+Ƭ,T
+ƭ,T
+Ʈ,T
+Ʋ,U
+Ƴ,Y
+ƴ,Y
+Ƶ,Z
+ƶ,Z
+ƿ,p
+Ǆ,DZ
+ǅ,DZ
+ǆ,DZ
+Ǉ,LJ
+ǈ,LJ
+ǉ,LJ
+Ǌ,NJ
+ǋ,NJ
+ǌ,NJ
+ǝ,E
+Ǣ,AE
+ǣ,AE
+Ǥ,G
+ǥ,G
+Ǳ,DZ
+ǲ,DZ
+ǳ,DZ
+Ƕ,H
+Ƿ,p
+Ǽ,AE
+ǽ,AE
+Ǿ,O
+ǿ,O
+Ƞ,n
+ȡ,d
+Ȥ,Z
+ȥ,Z
+ȴ,l
+ȵ,n
+ȶ,t
+ȷ,j
+ȸ,db
+ȹ,qp
+Ⱥ,A
+Ȼ,C
+ȼ,C
+Ƚ,l
+Ⱦ,T
+ȿ,s
+ɀ,z
+Ƀ,b
+Ʉ,U
+Ʌ,V
+Ɇ,E
+ɇ,E
+Ɉ,J
+ɉ,J
+Ɋ,Q
+ɋ,Q
+Ɍ,R
+ɍ,R
+Ɏ,Y
+ɏ,Y
+Ⱡ,L
+ⱡ,L
+Ɫ,L
+Ᵽ,P
+Ɽ,R
+ⱥ,A
+ⱦ,T
+Ⱨ,H
+ⱨ,H
+Ⱪ,K
+ⱪ,K
+Ⱬ,Z
+ⱬ,Z
+Ɑ,D
+Ɱ,M
+Ɐ,A
+Ɒ,D
+ⱱ,v
+Ⱳ,W
+ⱳ,W
+ⱴ,v
+ⱸ,e
+ⱺ,o
+ⱻ,E
+ⱼ,j
+ⱽ,v
+Ȿ,s
+Ɀ,z
+ẚ,a
+ẛ,f
+ẜ,f
+ẝ,f
+Ỻ,IL
+ỻ,IL
+Ỿ,V
+ỿ,V
```

Comment:
I've put in [library](https://github.com/xuender/unidecode) to replace our manual mapping. Most of the libraries are based on the same [perl implementation](http://search.cpan.org/~sburke/Text-Unidecode-0.04/lib/Text/Unidecode.pm) but I chose this one because it is on maven central.

It isn't perfect (none of these can be I think) but it's pretty good and better that the CSV.